relops_hardware_controller/settings.py

```diff
@@ -187,10 +187,13 @@ def CACHES(self):
         for task_name in TASK_NAMES.value
     ], seq_separator=',', environ_prefix=None)
 
-    VALID_WORKER_ID_REGEX = values.Value('^t-linux64-ms.*', environ_prefix=None)
+    VALID_WORKER_ID_REGEX = values.Value('^.*', environ_prefix=None)
 
     # Worker Settings
 
+    NOTIFY_EMAIL = values.Value('dhouse@mozilla.com', environ_prefix=None)
```

Comment:
We should make a groups list for this email.

Reply:
:+1:  I'll create a groups list for this (first I'll see if I have the perms to create it). Then I'll add a new patch in puppet to set it to use the groups list.